tests/sysprep_test.go

```diff
@@ -0,0 +1,361 @@
+    <RestartEnabled>true</RestartEnabled>
+    </component>
+    <component name="Microsoft-Windows-ehome-reg-inf" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="NonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <RestartEnabled>true</RestartEnabled>
+    </component>
+    </settings>
+    </unattend>
+    `
+)
+
+func insertProductKeyToAnswerFileTemplate(answerFileTemplate string) string {
+	productKeyFilePath := os.Getenv("KUBEVIRT_WINDOWS_PRODUCT_KEY_PATH")
+	keyFromFile, err := ioutil.ReadFile(productKeyFilePath)
+	Expect(err).ToNot(HaveOccurred())
+	productKey := strings.TrimSpace(string(keyFromFile))
+	return fmt.Sprintf(answerFileTemplate, productKey)
+}
+
+var getWindowsSysprepVMISpec = func() v1.VirtualMachineInstanceSpec {
+	gracePeriod := int64(0)
+	spinlocks := uint32(8191)
+	firmware := types.UID(windowsFirmware)
+	_false := false
+	return v1.VirtualMachineInstanceSpec{
+		TerminationGracePeriodSeconds: &gracePeriod,
+		Domain: v1.DomainSpec{
+			CPU: &v1.CPU{Cores: 2},
+			Features: &v1.Features{
+				ACPI: v1.FeatureState{},
+				APIC: &v1.FeatureAPIC{},
+				Hyperv: &v1.FeatureHyperv{
+					Relaxed:   &v1.FeatureState{},
+					VAPIC:     &v1.FeatureState{},
+					Spinlocks: &v1.FeatureSpinlocks{Retries: &spinlocks},
+				},
+			},
+			Clock: &v1.Clock{
+				ClockOffset: v1.ClockOffset{UTC: &v1.ClockOffsetUTC{}},
+				Timer: &v1.Timer{
+					HPET:   &v1.HPETTimer{Enabled: &_false},
+					PIT:    &v1.PITTimer{TickPolicy: v1.PITTickPolicyDelay},
+					RTC:    &v1.RTCTimer{TickPolicy: v1.RTCTickPolicyCatchup},
+					Hyperv: &v1.HypervTimer{},
+				},
+			},
+			Firmware: &v1.Firmware{UUID: firmware},
+			Resources: v1.ResourceRequirements{
+				Requests: k8sv1.ResourceList{
+					k8sv1.ResourceMemory: resource.MustParse("2048Mi"),
+				},
+			},
+			Devices: v1.Devices{
+				Disks: []v1.Disk{
+					{
+						Name:       windowsSealedDisk,
+						DiskDevice: v1.DiskDevice{Disk: &v1.DiskTarget{Bus: "sata"}},
+					},
+					{
+						Name:       "sysprep",
+						DiskDevice: v1.DiskDevice{CDRom: &v1.CDRomTarget{Bus: "sata"}},
+					},
+				},
+			},
+		},
+		Volumes: []v1.Volume{
+			{
+				Name: windowsSealedDisk,
+				VolumeSource: v1.VolumeSource{
+					Ephemeral: &v1.EphemeralVolumeSource{
+						PersistentVolumeClaim: &k8sv1.PersistentVolumeClaimVolumeSource{
+							ClaimName: tests.DiskWindowsSysprep,
+						},
+					},
+				},
+			},
+			{
+				Name: "sysprep",
+				VolumeSource: v1.VolumeSource{
+					Sysprep: &v1.SysprepSource{
+						ConfigMap: &k8sv1.LocalObjectReference{
+							Name: "sysprepautounattend",
+						},
+					},
+				},
+			},
+		},
+	}
+
+}
+
+const (
+	windowsSysprepedVMIUser     = "Admin"
+	windowsSysprepedVMIPassword = "Gauranga"
+)
+
+var _ = Describe("[Serial]Windows VirtualMachineInstance", func() {
```

Comment:
It would be good to include the focus string here with brackets, as we do for the SIGs, something like:
```suggestion
var _ = Describe("[Serial][Windows-sysprep] VirtualMachineInstance", func() {
```